guacamole/src/main/webapp/translations/en.json

```diff
@@ -447,6 +448,9 @@
         "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript name:",
         "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript path:",
 
+        "FIELD_OPTION_BACKSPACE_8"    : "Backspace (Ctrl-H)",
+        "FIELD_OPTION_BACKSPACE_127"  : "Delete (Ctrl-?)",
```

Comment:
These options are swapped compared to the telnet options. Which is correct?

Reply:
Fixed.  SSH is correct - ASCII 8 is "Backspace", ASCII 127 is Del.

http://www.physics.udel.edu/~watson/scen103/ascii.html

Reply:
You'll also need an entry for the `""` value (`FIELD_OPTION_BACKSPACE_EMPTY`).

Reply:
Added.